.linthtmlrc.json

```diff
@@ -1,3 +1,49 @@
 {
-  "extends": "@mate-academy/linthtml-config"
+  "attr-bans": [
```

Comment:
<img width="1427" alt="image" src="https://user-images.githubusercontent.com/38065883/173183495-1350180a-194d-4549-bb6f-2615adaf3c00.png">
It's better to use `flex-start`, so it the structure will remain if there are not enough cards on the page